attestation-agent/lib/src/lib.rs

```diff
@@ -12,12 +12,12 @@ use async_trait::async_trait;
 use attester::{detect_tee_type, BoxedAttester};
 use kbc::{AnnotationPacket, KbcCheckInfo, KbcInstance, KbcModuleList};
 use resource_uri::ResourceUri;
-use std::collections::HashMap;
+use std::{collections::HashMap, path::Path};
 
-#[cfg(feature = "cc_kbc")]
-mod token;
-#[cfg(feature = "cc_kbc")]
-use token::get_kbs_token;
+mod config;
+
+#[cfg(any(feature = "coco_as", feature = "kbs_as"))]
+use token::GetToken;
```

Comment:
Maybe we could have an intermediate feature named `_token` in `Cargo.toml`.
```toml
# Whether token provisioning is supported by AA
_token = []

coco_as_token = ["token/coco_as", "_token"]
kbs_token = ["token/kbs", "_token"]
```

And then we can here
```rust
#[cfg(feature = "_token")]
use token::GetToken;
```